tests/.env.example

```
@@ -34,8 +34,14 @@
-# SOLUTION_SERVER_PASSWORD='Dog8code'
+#####################################
+### SOLUTION SERVER CONFIGURATION ###
+#####################################
+
+# SOLUTION_SERVER_URL="https://your-solution-server.com"
+# SOLUTION_SERVER_USERNAME="username"
+# SOLUTION_SERVER_PASSWORD="password"
+# SOLUTION_SERVER_REALM="your-realm"
+# SOLUTION_SERVER_INSECURE=true
+# SOLUTION_SERVER_LOCAL=false
+# LOCAL_MCP_TOKEN="your-local-mcp-token"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)